cmd/crdvalidator/handlers/crd.go

```diff
@@ -74,12 +75,6 @@ func (cv *CrdValidator) Handle(ctx context.Context, req admission.Request) admis
 	return admission.Allowed("")
 }
 
-// InjectDecoder injects a decoder for the CrdValidator.
-func (cv *CrdValidator) InjectDecoder(d *admission.Decoder) error {
-	cv.decoder = d
-	return nil
-}
-
```

Comment:
Any particular reason for this removal? I noticed that this made the go-apidiff check unhappy

Reply:
That was the old way controller-runtime could inject a Decoder. That interface was removed, and the function is no longer useful.

I updated the code to set the decoder in the constructor function instead.